org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/ClassFile.java

```diff
@@ -5764,6 +5764,16 @@ public byte[] getBytes() {
 		}
 		return this.bytes;
 	}
+
+	/**
+	 * Sets the actual bytes of the class file.
+	 *
+	 * This method can be used e.g. to store the modified byte representation of the class file after post processing.
+	 */
```

Comment:
It is not API, but nevertheless one should add a note that "this method is made public only to be accessible from AbstractImageBuilder during compilation post processing. Using this method for any other purpose is discouraged and may lead to unpredictable results".